packages/shared/src/components/profile/Header.tsx

```diff
@@ -5,22 +5,20 @@ import { ShareIcon, SettingsIcon } from '../icons';
 import { Button, ButtonSize, ButtonVariant } from '../buttons/Button';
 import { useShareOrCopyLink } from '../../hooks/useShareOrCopyLink';
 import { ProfilePicture } from '../ProfilePicture';
-import { largeNumberFormat } from '../../lib/numberFormat';
+import { largeNumberFormat } from '../../lib';
 import { ReferralCampaignKey } from '../../lib/referral';
```

Comment:
This gives a dep loop if you do both from lib?

Reply:
There was none, so I think it's safe.

Reply:
I mean to move both to import from `lib` so without the `referral` part